lib/linux_admin/fstab.rb

```diff
@@ -44,30 +44,35 @@ def columns
     end
 
     def column_lengths
-      self.columns.collect { |c| c ? c.size : 0 }
+      columns.collect { |c| c ? c.to_s.size : 0 }
     end
 
     def formatted_columns(max_lengths)
       self.columns.collect.
-        with_index { |col, i| col.to_s.rjust(max_lengths[i]) }.join(" ")
+        with_index { |col, i| col.to_s.rjust(max_lengths[i]) }.join(" ").rstrip
     end
   end
 
   class FSTab
     include Singleton
 
-    attr_accessor :entries
-    attr_accessor :maximum_column_lengths
-
     def initialize
       refresh
     end
 
+    def entries
+      @entries ||= LinuxAdmin::FSTab::EntryCollection.new
+    end
+
+    def maximum_column_lengths
+      entries.maximum_column_lengths
+    end
```

Comment:
Do we care about the setter methods that we're losing by implementing only the getter method?

Reply:
I don't even know what the setter would do in this case ... Nothing productive for sure.

Reply:
I don't think the setter should have ever been public api